vendor/github.com/vmware/govmomi/toolbox/command.go

```diff
@@ -472,19 +473,22 @@ func (c *CommandServer) ListFiles(header vix.CommandRequestHeader, data []byte)
 		return nil, err
 	}
 
-	info, err := os.Stat(r.GuestPathName)
+	info, err := os.Lstat(r.GuestPathName)
```

Comment:
In general, yes.  But this change (and the others I believe), are from govmomi upstream.  Just needs a rebase (or gvt update).